src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -945,13 +945,15 @@ internal static uint ResetLowestSetBit(uint value)
         }
 
         /// <summary>
-        /// Reset specific bit in the given value
+        /// Flip the bit at a specific position in a given value.
+        /// Similar in behavior to the x86 instruction BTC (Bit Test and Complement).
         /// </summary>
+        /// <param name="value">The value.</param>
+        /// <param name="index">The zero-based index of the bit to flip.</param>
```

Comment:
do we really need to mark this aggressive inlining, surely such a small method will be very likely inlined?

Reply:
It's within the "always inline" limit for CoreCLR (less than 16-bytes of IL, the old implementation was notably under this as well).

I'm uncertain if it will be impactful to Mono, where the inliner works quite a bit differently.

Reply:
The method is compiled to 9-bytes of IL, so it would almost certainly be inlined.

However, there are many small methods in `BitOperations` that have this attribute (including `ResetBit` that it replaced), so my suggestion is to maintain consistency in this PR.

Reply:
cc;@AndyAyersMS

Reply:
For RyuJit it doesn't matter one way or the other. Since there's a consistent pattern in the sources, we might as well continue to follow it for now.